site/_docs/reference.md

```diff
@@ -2752,6 +2752,8 @@ In the following:
 | * | ATANH(numeric)                                 | Returns the inverse hyperbolic tangent of *numeric*
 | f | BITAND_AGG(value)                              | Equivalent to `BIT_AND(value)`
 | f | BITOR_AGG(value)                               | Equivalent to `BIT_OR(value)`
+| * | BITCOUNT(integer)                             | Returns the bitwise COUNT of non-null *integer*
```

Comment:
Then the test case maybe should include a negative number and zero.

Reply:
In MYSQL:
```
select bit_count(3.5);  return 1
select bit_count(3); return 2
select bit_count(5.5); return 2
```
So support float number?

Reply:
In calcite these are decimal numbers

Reply:
Since the decimal representation of a number is not specified, how can these functions be implemented?

Reply:
The documentation has been updated. If it makes more sense, both `BITCOUNT()` and `BIT_COUNT()` can use the MySQL behaviour and become identical.

Reply:
Are you sure Spark also has a bigcount function? I didn't find the corresponding record in the documentation

Reply:
`BIT_COUNT` is now enabled for Apache Spark.